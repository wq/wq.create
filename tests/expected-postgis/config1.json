{
    "pages": {
        "login": {
            "url": "login",
            "name": "login"
        },
        "logout": {
            "url": "logout",
            "name": "logout"
        },
        "index": {
            "url": "",
            "name": "index",
            "show_in_index": false,
            "verbose_name": "test Project",
            "map": {
                "mapId": "map",
                "layers": []
            }
        }
    },
    "site_title": "test Project",
    "router": {
        "base_url": ""
    },
    "store": {
        "service": "",
        "defaults": {
            "format": "json"
        }
    },
    "logo": "/static/app/images/icon-192.png",
    "material": {
        "theme": {
            "primary": "#7500ae",
            "secondary": "#0088bd"
        }
    },
    "map": {
        "bounds": [
            [
                -180,
                -70
            ],
            [
                180,
                70
            ]
        ],
        "tiles": "/tiles/{z}/{x}/{y}.pbf"
    },
    "debug": true
}
