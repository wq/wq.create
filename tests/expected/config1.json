{
    "pages": {
        "login": {
            "url": "login",
            "name": "login"
        },
        "logout": {
            "url": "logout",
            "name": "logout"
        },
        "index": {
            "url": "",
            "name": "index",
            "show_in_index": false,
            "verbose_name": "test Project"
        }
    },
    "site_title": "test Project",
    "router": {
        "base_url": ""
    },
    "store": {
        "service": "",
        "defaults": {
            "format": "json"
        }
    },
    "material": {
        "theme": {
            "primary": "#7500ae",
            "secondary": "#0088bd"
        }
    },
    "map": {
        "bounds": [
            [
                -180,
                -70
            ],
            [
                180,
                70
            ]
        ]
    },
    "debug": true
}
